rubocop.yml

```diff
@@ -1907,11 +1907,11 @@ Metrics/ModuleLength:
   Max: 100
 
 Metrics/ParameterLists:
-  Description: 'Avoid parameter lists longer than 2 parameters. Use keyword args instead.'
+  Description: 'Avoid parameter lists longer than 3 parameters. Use keyword args instead.'
   StyleGuide: '#too-many-params'
   Enabled: true
   Severity: error
-  Max: 7
+  Max: 3
```

Comment:
I feel like 3 is waaaay to short.
How about 5 to start with? We have many functions with more than 3 i believe